htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -373,6 +373,15 @@ UPDATE llx_propal SET date_signature = date_cloture WHERE date_signature IS NULL
 
 
 ALTER TABLE llx_product ADD COLUMN batch_mask VARCHAR(32) NULL;
+ALTER TABLE llx_product ADD COLUMN life_duration VARCHAR(6);
+ALTER TABLE llx_product ADD COLUMN control_periodicity VARCHAR(6);
+
+ALTER TABLE llx_product_lot ADD COLUMN eol_date datetime;
+ALTER TABLE llx_product_lot ADD COLUMN manufacturing_date datetime;
+ALTER TABLE llx_product_lot ADD COLUMN scrapping_date datetime;
+ALTER TABLE llx_product_lot ADD COLUMN commissionning_date datetime;
+ALTER TABLE llx_product_lot ADD COLUMN control_periodicity VARCHAR(6);
```

Comment:
Can you rename in 
qc_frequency   (Quality control Frequency)
I suppose this is what you mean.

Reply:
that’s exactly it means, I’ll do it